blog/electron-33-0.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Electron 33.0.0
+date: 2024-10-14T00:00:00.000Z
+authors: ckerr
+slug: electron-33-0
+tags: [release]
+---
+
+Electron 33.0.0 has been released! It includes upgrades to Chromium 130.0.6723.44, V8 13.0, and Node 20.18.0.
+
+---
+
+The Electron team is excited to announce the release of Electron 33.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Highlights
+
+- Added an event to help unlock cryptographic devices when a PIN is needed. [#41205](https://github.com/electron/electron/pull/41205)
```

Comment:
The release notes for this PR are unfortunately wrong.  I think the original intention was to add an event, but what was instead added was a handler, `app.setClientCertRequestPasswordHandler(handler)`, eg see https://github.com/electron/electron/pull/41205/files#diff-328405e92e0015c64835b31b2679ebac5faf15d70c12d5fe812211edaea01e8eR1492.

Reply:
updated in ed6a3471798bf437821f7b97b08e0cb307739700

Reply:
updated in ed6a3471798bf437821f7b97b08e0cb307739700